core/templates/dev/head/pages/exploration_player/conversation_skin_embed_directive.html

```diff
@@ -429,6 +429,10 @@
     border-bottom-right-radius: 0;
   }
 
```

Comment:
Ditto here as well.

Reply:
I have added a comment block to explain the rationale behind the CSS block.